tutorials/examples/mRNA_design/MFE_calculator.py

```diff
@@ -0,0 +1,830 @@
+import math
```

Comment:
this file should be broken in different pieces. E.g. the TURNER_FE values can go in a json or in a python file itself. Utilities function can also go in another file